Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+            padding-left:10px;  
+         }
+         ul .kutu{
+            display: flex;
+            align-items: flex-start;
+         } 
+         .fa-circle-check{
+         color: red;
+         }
+         .form-input{
+            width: 100%;
+            height: 35px;
+            border-radius: 5px;
+            margin-bottom: 20px;
+         }
+         button{
+            width:calc(100% + 10px);
+            height: 50px;
+            background-color: rgb(4, 4, 30);
+            color: white;
+            padding: 5px auto;
+            border-radius: 10px;
+            cursor: pointer; 
+         }
+         button:hover{
+            background-color: #590808;
+            transition: .5s;
+         }
+        .error {
+            color: #590808;
+            font-size: 12px;
+            margin-top: 5px;
+            margin-bottom: 10px;
+            display: flex; 
+            justify-content: flex-end;  
+        }
+        .sayfa2{
+         display: none;
+        }
+       
+
+       
+    </style>
+</head>
+<body>
+    <div class="kapsayici">
+        
+       <div class="anaMenu">
+        <div class="solEkran">
+            <h2>Stay Updated!</h2>
+            <p>Join 60.0004 product managers receiving monthly updates on:</p>
+           <ul>
+               <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>Product discovery and building what matters</li></div>  
+                <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>Measuring to ensure updates are success </li></div>
+                <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>And much more!</li></div>
+            </ul>
+            <form id="subscribeForm" novalidate>
+            <label for="email" class="form-label" >Email Address:</label>
+            <div id="emailError" class="error"></div>
+            <div id="emailError1" class="error1"></div>
+            <input type="email" id="email" name="email" class="form-input" placeholder="Email@company.com"required>
+            <div class="buton"><button>Subscribe to monthly newsletter</button></div></form>
+        </div>
+        <div class="sagEkran">
+            <img src="./1.png" alt="resim1">
+        </div>
+        </div>
+    <div class="sayfa2"><img src="./index2.html"></div>
+    </div>
+    
+    <script>
+        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+        
+        const sayfa2 =document.getElementsByClassName("sayfa2");
+        const form = document.getElementById("subscribeForm");
+        form.addEventListener("submit",function(event){
+            event.preventDefault();
+         let isFormCorrect =true;
+        const emailInput = document.getElementById("email");
+        const emailError = document.getElementById("emailError");
+if (!emailRegex.test(emailInput.value)){
+    emailError.textContent='Valid email required.';
+   isFormCorrect=false;
+}  
+ else if(emailRegex.test(emailInput.value)){
```

Comment:
Remove this `else if` case. Instead, you'd need to create a class 'hidden' in CSS and add/remove it to either div with the form or div with confirmation message.